.circleci/config.yml

```diff
@@ -84,7 +84,7 @@ commands:
           name: Run Integration Tests
           no_output_timeout: 60m
           command: |
-            go test -tags=integration -timeout=60m -v -p 1 ./test/integration/...
+            go test -tags=integration -timeout=60m -v -p 1 ./test/integration/http/...
```

Comment:
@nicob87 guess this needs to be restored

Reply:
correct!